lib/audited/auditor.rb

```diff
@@ -141,15 +139,11 @@ def audited_attributes
       protected
 
       def revision_with(attributes)
+        instance_variables = { "new_record" => self.destroyed?, "persisted" => !self.destroyed?, "readonly" => false, "destroyed" => false, "_destroyed" => false, "marked_for_destruction" => false }
```

Comment:
Yeah I agree, I did this nearly 12 months ago and have learnt a lot more since then. Thanks for the feedback!